### DISCUSSION TITLE
Prepare for First Release

### DISCUSSION THREAD

**Discussion by aljawaid** [creecros/Mailmagik#9] [General]
- [x] Make plugin functional - @creecros @alfredbuehler 
- [ ] Style pages for frontend - anybody or @aljawaid 
- [ ] Add translations to plugin - anybody or @aljawaid (can make a empty string translation template from KSP Template)
- [ ] Update readme with clear description, features and installation etc - anybody or @aljawaid 
- [ ] Rename Repository, possible rename plugin (just the folder name and plugin name) - can be done by @aljawaid 
- [ ] Publish release - anybody or @aljawaid 
- [ ] Submit to directory - anybody or @aljawaid

**Comment by aljawaid** [comment]
no deadlines, just a structure of what needs doing

**Reply by creecros** [reply]
well, it is definitely functional, so check that box

**Comment by creecros** [comment]
@aljawaid 

I'm all set from a functional standpoint. if there is anything else you want to do before I package for release, now is your chance.

**Reply by aljawaid** [reply]
I will look at it again tonight and let you know. Have you done the docs/screenshots?

**Reply by aljawaid** [reply]
Do I need to look at the template/css or have you sorted it?

**Comment by aljawaid** [comment]
I suggest changing the name. TaskEmailer or something more meaningful

**Reply by creecros** [reply]
ook, I thought really long and hard about it, and I came up with kbphpimap

:smile:

**Reply by aljawaid** [reply]
> ook, I thought really long and hard about it, and I came up with kbphpimap
> 
> :smile: 

Hahaha the most confusing name ever hahahaha the current name is developer attracting but not user friendly. Nobody would even think of downloading the plugin hahaha

**Reply by creecros** [reply]
I think the description will prevail when people search,  and IMAP is kind of the most important part. I could see shortening it to kbimap and drop the php, but meh. they will find it, whoever they are. better than never publishing it and sitting on it for another 2 years

**Reply by alfredbuehler** [reply]
Can't you achieve this by setting the appropriate topics in the repo? E.g. Kanboard, Plugin, IMAP, Productivity

**Reply by creecros** [reply]
Probably, i'll change the name after you push your parsing feature, don't want to screw you up with your merge

**Reply by alfredbuehler** [reply]
Go ahead, buddy. It's done. ;)
